doc/introduction/data.rst

```diff
@@ -90,10 +90,10 @@ been generated for formic acid at zero temperature. The GBS samples can be used
 .. autosummary::
     Formic
 
-Dataset
--------
+SampleDataset
+-------------
 
-The :class:`Dataset` class provides the base functionality from which all datasets inherit.
+The :class:`SampleDataset` class provides the base functionality from which all datasets inherit.

```

Comment:
```suggestion
The :class:`~.SampleDataset` class provides the base functionality from which all datasets inherit.
```
The tilde and dot mean that it will search for a class with name `SampleDataset` :+1: